flinkdatasource/src/main/java/org/zuinnote/flink/bitcoin/BitcoinRawBlockFlinkInputFormat.java

```diff
@@ -32,54 +32,50 @@
 
 
 public class BitcoinRawBlockFlinkInputFormat extends AbstractBitcoinFlinkInputFormat<BytesWritable> implements CheckpointableInputFormat<FileInputSplit, Long> {
-	
+
 
 
 	private static final Log LOG = LogFactory.getLog(BitcoinRawBlockFlinkInputFormat.class.getName());
 	/**
-	 * 
+	 *
 	 */
 	private static final long serialVersionUID = 4150883073922261077L;
 	private boolean isEndReached;
-	
+
 	public BitcoinRawBlockFlinkInputFormat(int maxSizeBitcoinBlock, String specificMagicStr,
-			boolean useDirectBuffer) throws HadoopCryptoLedgerConfigurationException {
+										   boolean useDirectBuffer) throws HadoopCryptoLedgerConfigurationException {
 		this(maxSizeBitcoinBlock, specificMagicStr, useDirectBuffer,false);
-	
+
 	}
-	
+
 	public BitcoinRawBlockFlinkInputFormat(int maxSizeBitcoinBlock, String specificMagicStr,
-			boolean useDirectBuffer, boolean readAuxPOW) throws HadoopCryptoLedgerConfigurationException {
+										   boolean useDirectBuffer, boolean readAuxPOW) throws HadoopCryptoLedgerConfigurationException {
 		super(maxSizeBitcoinBlock, specificMagicStr, useDirectBuffer,readAuxPOW);
 		this.isEndReached=false;
 	}
-	
-	@Override
```

Comment:
same comment as for the other class, I don't think we can move it to AbstractBitcoinFlinkInputFormat

Reply:
Acknowledged.